src/Shared/Internal/HttpContextExtensions.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+#if !ASP_NET_CORE
+using System.Web;
+#else
+using System.Text;
+using Microsoft.AspNetCore.Http;
+#endif
+using NLog.Common;
+
+namespace NLog.Web.Internal
```

Comment:
Any idea why this isn't marked as move? Changed so much?

Reply:
Think it changed too much

Reply:
Could you please add a commit before this with only the rename? Or could I do that?

Reply:
Now two separate commits

Reply:
thanks!